docs/components/concepts/data-retention.md

```diff
@@ -0,0 +1,22 @@
+---
+id: data-retention
+title: "Data retention"
+---
+
+In Camunda Platform 8 SaaS, the following data retention strategies are implemented. This is necessary as the amount of data can grow significantly over time. These settings are a balance between performance and usability.
+
+## Retention time of each application
+
+The following time-to-live settings are configured in SaaS for each application.
```

Comment:
Could we add three things:
- These are defaults, please reach out if you have specific requirements
- These are defaults for Enterprise Plan
- Data Retention can be configured for Self-Managed.

Reply:
Thanks for the input. The defaults for an enterprise plan are the same right?

Reply:
I meant that I think we should for now call them "defaults for Enterprise" as we still want to consider changing defaults for Prof and Trial. So we can add later add information about Prof and Trial

Reply:
Added the mentioned things.